### DISCUSSION TITLE
Feature request : Allow import of non-module CSS files directly into components (+1)

### DISCUSSION THREAD

**Discussion by rahul3v** [vercel/next.js#36014] [Ideas]
It's batter to have **import of non-module CSS files directly into components**, as some times we don't need `.module.css` files,

So **NextJs must add this option** too, every other popular frameworks didn't remove that feature, as it's useful many times.

As `styled-jsx` styling option is painful as **it doesn't provide auto-complete options**

**Give +1 if you need this feature in NextJs**

**Comment by Babailan** [comment]
CSS Modules are an optional feature and are only enabled for files with the `.module.css` extension.
you can add global css as many as you want. `.module.css` is a indicator for solo component styles and it's also powerful tho
Regular <link> stylesheets and global CSS files are still supported.

**Reply by rahul3v** [reply]
I am talking about directly adding css file **at component level**, not about `.module.css`
```
import "./component_styles.css";
```

**Reply by Babailan** [reply]
@rahul3v It's a little bit complicated because it's a Component which means that it'll be imported `.module.css` from its pages that's why. CSS module is creating random className so if you just `import "./component_styles.css";` the scope of CSS module will break.

as you can see in the image the className there is 0lcS2 generated after the name of className.

![image](https://user-images.githubusercontent.com/83863770/162487273-2337e637-cd20-49d0-be90-0aad98f48122.png)

I think that's possible but idk It's a  good idea tho.

**Reply by rahul3v** [reply]
sometimes we don't need that, to avoid writing module based classname syntexes multiple times, it gives flexibility to users as per their project structure
